lib/screens/map_view.dart

```diff
@@ -61,11 +62,14 @@ class _MapViewScreenState extends State<MapViewScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-          title: Center(child: Text(widget.title)),
-          automaticallyImplyLeading: false),
+        centerTitle: true,
+        title: Image.asset('assets/images/logos_and_icons/logoshrink-removebg.png', height: 40),
```

Comment:
If we are going to reuse this logo in multiple places, can we make it a global variable we store somewhere?

Reply:
That's a good idea, it would definitely make it easier to use

Reply:
Do you want me to make that change before approval?